Code/RDGeneral/Dict.h

```diff
@@ -170,10 +170,18 @@ class RDKIT_RDGENERAL_EXPORT Dict {
       }
     }
     throw KeyErrorException(what);
-  }
+  };
```

Comment:
The semi colon isn't necessary.   Mildly surprised this isn't an error.

Also in the function above here.

Reply:
That's legal syntax (and is used pretty regularly in the RDKit), but it's not needed.
https://stackoverflow.com/questions/9997895/semicolon-after-function

I'll remove it here and will try to remember to remove those trailing semicolons in the future.

Reply:
A semicolon after a function (non class) only became legal in C++11 I think.  It doesn't really matter except for consistency and as long as we don't start seeing:

```
void foo() {
};;;;;;;;; // Hi Brian!
```